tests/test_ping_serve/test_ping_serve.c

```diff
@@ -317,6 +319,10 @@ static void ping_serve(int verbose)
         if (solo5_net_write_sync(buf, len) == -1)
             puts("Write error\n");
 
+        received++;
+        if (received == 100000)
```

Comment:
Can you add a message here saying that we are exiting because of this limit please.

It would be nice to pass this number as a parameter as I usually test virtio-net by letting this run for a while. Do you mind if i create a PR afterwards making this an arg? (unless you want to do it as part of this PR)